### DISCUSSION TITLE
please give me simple chinese vision xournal++.

### DISCUSSION THREAD

**Discussion by lhbadsl** [xournalpp/xournalpp#2696] [Q&A]
I am an xournal++ user from China. I have advanced the xournal++ simplified Chinese translation project to 97% some time ago, but there are a few sentences that I really can’t translate. I have tried my best to recommend xournal++ among Chinese friends and try to get them The translation of xournal++, but it seems to have no effect for the time being. I will continue to promote the simplified Chinese version of xournal++. I need the simplified Chinese version. I hope the official can add the Chinese version as soon as possible. I like this software so much that my Work cannot do without it.

**Comment by rolandlo** [comment]
What exactly do you mean by "it seems to have no effect for the time being"? Could you post a screenshot of how Xournal++ currently looks in simplified Chinese version? Which version of Xournal++ do you use? Probably you need version 1.1.0+dev. And what operating system do you use?

**Reply by lhbadsl** [reply]
Sorry, I don’t know English very well, but I can still translate some sentences, because I am a user of xournal++, so I can guess the specific meaning of English; I mean I recommend xournal++ among my friends around and call on them to translate , There is no effect. The system I am using is UOS, and there is no Chinese version on it. I am currently using the English version.

**Comment by lhbadsl** [comment]
my xournal++ vision is 1.0.20-hotfix

**Comment by rolandlo** [comment]
In Xournal++ 1.1.0+dev it looks like this: 
![grafik](https://user-images.githubusercontent.com/40485433/105577302-f868ab80-5d78-11eb-867b-451777dd7419.png)

**Reply by lhbadsl** [reply]
thank you,i will try it.

**Comment by rolandlo** [comment]
The download link for the nightly release of version 1.1.0+dev is [here](https://github.com/xournalpp/xournalpp/releases/tag/nightly). You may need the AppImage, since the fix of the .deb version is not yet ported to version 1.1.0+dev. Please check it out and report back, if it works.

**Reply by lhbadsl** [reply]
thank you so much!OK,i will report back.

**Reply by lhbadsl** [reply]
I have used the 1.1.0dev version. I will improve the translation during use and recommend it to friends around me. Thank you for your efforts, so that we can have such a good note-taking application.

**Comment by rolandlo** [comment]
Ok, great! Note that you can see when the translations get updated from "New Crowdin updates" in [the commit list](https://github.com/xournalpp/xournalpp/commits/master)
